websites/S/Shiroko/presence.ts

```diff
@@ -0,0 +1,107 @@
+import { getEpisode, getId, getImage, getTitle } from "./utils";
+
+const presence = new Presence({
+	clientId: "1286500193881161804",
+});
+
+let lastPlaybackState = false,
+	playback = false,
+	startTimestamp = Math.floor(Date.now() / 1e3),
+	lastHref: string,
+	lastUpdate = 0;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			buttons: [{ label: "Visit Shiroko", url: "https://shiroko.co" }],
```

Comment:
Buttons which direct to the main page are [prohibited](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets).
```suggestion
```